.github/workflows/dotnet-format.yml

```diff
@@ -20,6 +20,11 @@ jobs:
     - name: check out code
       uses: actions/checkout@v3
 
+    - name: Install .NET 7 SDK
```

Comment:
Do we need this? This was not required before.

Reply:
Seems to be needed for the moment. In global.json we now require 7.0.101. The reason for that is 7.0.100 has a bug where code analysis doesn't run. When I bumped it to 101 then VS started running the same analysis as the CI. Problem is, dotnet format workflow was still using 100. Once I added this it started pulling down 101. My guess would be 101 is still rolling out to all the images?